Clicker/Views/Cards/NavigationTitleView.swift

```diff
@@ -61,11 +61,11 @@ class NavigationTitleView: UIView {
         addSubview(navigationButton)
     }
     
-    func configure(primaryText: String, secondaryText: String, delegate: NavigationTitleViewDelegate? = nil) {
+    func configure(primaryText: String, secondaryText: String, userRole: UserRole? = nil, delegate: NavigationTitleViewDelegate? = nil) {
```

Comment:
will userRole ever be `nil`? if not, then we don't need to make it an optional param

Reply:
The reason for it being optional is that configure for NavigationTitleView was being called in other parts of the CodeBase where userRole wasn't in that view controller.

Reply:
ah ok gotcha